.github/ISSUE_TEMPLATE/repo-permissions.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Repository Maintenance Request
+about: Request changes on an existing OpenTelemetry repository
+title: 'REQUEST: Repository maintenance on <REPOSITORY-NAME>'
+labels: area/repo-maintenance
+assignees: ''
+---
+
+<!-- Please remember to change the title of this issue by replacing
+ <REPOSITORY-NAME> with the actual respository name -->
+
+### Affected Repository
+
+e.g. https://github.com/open-telemetry/community
+
+### Requested changes
+
+e.g. Give 'Admin' permissions to `@username`
```

Comment:
Maybe give another example for a one-time change that doesn't require ongoing permission?

e.g. Enable branch protection rule for `release-*`